### ISSUE TITLE
Windows binaries outdated

### ISSUE THREAD

**Issue by schlamar** [zealdocs/zeal#948]
The binaries for Windows are version 0.5 (which has some annoying bugs like #851). Please update them to the current version 0.6.

**Comment by STaRDoGG** [comment]
+1. I just came by to check and noticed no Windows binaries for 0.6.0. I personally use the Portable x64 version.

**Comment by naxvog** [comment]
You can use the CI Versions (sort by updated):
https://bintray.com/zealdocs/windows-ci/zeal-unstable/#files/zeal-unstable

**Comment by techtonik** [comment]
Website version is also outdated 0.5.0 - https://zealdocs.org

**Comment by bantya** [comment]
Yes,  the page https://github.com/zealdocs/zeal/releases/tag/v0.6.0 points to zeal website to download latest binaries but, the binary version is 0.5.0.

**Comment by nishanthkarthik** [comment]
@trollixx When can the Windows binary for 0.6.x be pushed for release? Is there anything blocking its release?
